packages/koa/typedoc.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "https://typedoc.org/schema.json",
+  "entryPoints": ["./src/exports.ts"],
+  "out": "docs",
+  "theme": "default",
+  "name": "@bitovi/scaffold",
+  "includeVersion": true,
+  "disableSources": true,
+  "githubPages": true,
+  "media": "./media",
+  "navigationLinks": {
+    "Bitovi": "https://bitovi.com",
+    "GitHub": "https://github.com/bitovi/bitscaffold"
```

Comment:
Should this be updated? Do we need to create setup an NX command to generate these docs?